src/js/cookie-policy.js

```diff
@@ -12,9 +12,20 @@ if (ubuntu.hasOwnProperty('cookiePolicy')) {
 // The cookie policy injection and interaction
 ubuntu.cookiePolicy = function() {
   let context = null;
+  let options = {
+    'content':
+      `We use cookies to improve your experience. By your continued
+      use of this site you accept such use. To change your settings
+      please
+      <a href="https://www.ubuntu.com/legal/terms-and-policies/privacy-policy#cookies">
+        see our policy
+      </a>.`
+  }
 
   return {
-    setup: function(content) {
+    setup: function(options) {
+      let content = options.content;
+      let duration = options.duration;
       let start = `
         <div class="p-notification--cookie-policy">
           <p class="p-notification__content">`;
```

Comment:
The ```.js-close``` element should be a ```<button>``` element as we're not linking to a new page or anchor on the page.